### PR TITLE
Spacemacs badge usage

### PR DESCRIPTION
**Pull request by nashamri**
After many trials and errors, I think this is the best way to display the badge and use it.
I couldn't get the SVG to show up with relative paths so I had to hardcode it :smile:

### CODE REVIEW

**Comment by d12frosted** [comment]
:+1: I also don't know any other way to display svg here.

**Comment by nashamri** [comment]
It seems to be a known issue https://github.com/isaacs/github/issues/316

**Comment by CestDiego** [comment]
Finally it seems you figured it out! :D I'm happy and I'm using it already https://github.com/CestDiego/sweetgreen.el

**Comment by CestDiego** [comment]
@nashamri I think it'd be better to have `https://github.com/syl20bnr/spacemacs` so that people can click on it and go to spacemacs page :P

**Comment by nashamri** [comment]
@CestDiego oh good catch!

**Comment by CestDiego** [comment]
@nashamri also..don't forget to give instructions on how to use it in Org-Mode:

```
[[https://github.com/syl20bnr/spacemacs][file:https://cdn.rawgit.com/syl20bnr/spacemacs/develop/assets/spacemacs-badge.svg]]
```

**Comment by nashamri** [comment]
Done! Thanks @CestDiego!

**Comment by robbyoconnor** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -269,7 +269,28 @@ bugs, helping the community on the [Gitter Chat][] or sending pull requests.
...
 
-Thank you !
+If you used spacemacs in a project and you want to show that fact, you can use
+the spacemacs badge ![https://github.com/syl20bnr/spacemacs](https://cdn.rawgit.com/syl20bnr/spacemacs/develop/assets/spacemacs-badge.svg).
+
+- For Markdown:
+
+   ```
+   ![https://github.com/syl20bnr/spacemacs](https://cdn.rawgit.com/syl20bnr/spacemacs/develop/assets/spacemacs-badge.svg)
+   ```
```
Comment (robbyoconnor) [inline README.md]: This is wrong, here's the corrrect version:
```markdown
[![Made with Spacemacs](https://cdn.rawgit.com/syl20bnr/spacemacs/develop/assets/spacemacs-badge.svg)](http://github.com/syl20bnr/spacemacs)

```

**Comment by robbyoconnor** [comment]
Added to the top in #3784 :heart:

**Comment by syl20bnr** [comment]
Thank you for this beautiful badge :100: 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**